#Ambient light message
Header header
uint16 lux 